contrib/test/integration/main.yml

```diff
@@ -64,6 +66,7 @@
       include: "build/kubernetes.yml"
       vars:
           force_clone: True
+          version: "cri-o-patched-1.9"
```

Comment:
Do we need the same PR for 1.8 branch as well with this changed to the 1.8 kube branch?

Reply:
task-level variables have a medium-ish [level of scoping](http://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-scopes).  They can be overridden at runtime from the ``vars.yml`` file, or on the ansible-command-line.  This is fine for now.

FWIW/FYI:  if this (or other include-files) grow in their use of variables,  that's a good sign they should be converted into roles (easy to do).  Roles allow setting these kinds of variables as 'defaults' - which have the absolute lowest-level of scoping.  This is important, because having that flexibility better supports complex situations, like a matrix of hosts with a combination of versions from multiple sources.